mentorship ios/Managers/KeyboardManager.swift

```diff
@@ -0,0 +1,27 @@
+////
+////  KeyboardManager.swift
+////  mentorship ios
+////
+////  Created by Yugantar Jain on 09/06/20.
+////  Copyright © 2020 Yugantar Jain. All rights reserved.
+////
+//
+//import Foundation
+//import Combine
+//import SwiftUI
+//
+//final class KeyboardManager: ObservableObject {
```

Comment:
remove this file

Reply:
This file will surely be used later. So I'd prefer to keep it.